### PR TITLE
Added en text to kuma blog

### PR DESCRIPTION
**Pull request by windsonsea**
为 Kuma blog 新增英文

### CODE REVIEW

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Kuma 2.0 integrated with Merbridge and improved the network latency by up to 12%
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
title: Merbridge helps Kuma reduce network latency by 12%
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+linkTitle: Kuma 2.0 integrated with Merbridge and improved the network latency by up to 12%
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
linkTitle: Merbridge helps Kuma reduce network latency by 12%
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+![Performance comparison between eBPF and iptables for Kuma 2.0](./imgs/kuma-2.0-ebpf-vs-iptables.png)
+
+A quote from [Kuma 2.0 release blog](https://kuma.io/blog/2022/kuma-2-0-0/):
+
+> We are utilizing the Merbridge OSS project within our eBPF capabilities and are very excited
+> that we have been able to contribute back to that library and become co-maintainers. We look forward to working more
+> with the Merbridge team as we continue to explore different areas to include eBPF functionality in Kuma.
+
+As a developer of Merbridge, I'm very glad that Merbridge empowers service meshes like Kuma.
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
As a developer of Merbridge, I'm very glad that Merbridge empowers service mesh providers like Kuma.
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+This case means the traffic latency can be improved without any extra overhead when you choose to use Merbridge in a service mesh.
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
This case proves that traffic latency can be reduced without any extra overhead if you use Merbridge in a service mesh.
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Since June this year, Kuma developers have been working on integrating with Merbridge, trying to get the eBPF-based acceleration capabilities.
+
+Thanks to the clear architecture of Merbridge, Kuma is smoothly adapted with Merbridge in days.
+A big thanks to the Kuma community for contributing such an important compatibility capability to Merbridge, which helps both communities grow together!
+
+So far, Merbridge has the capabilities to support for popular service meshes like Istio, Linkerd2, and Kuma,
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
So far, Merbridge has the capabilities to support popular service mesh products like Istio, Linkerd2, and Kuma,
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+and also has a clear plan to develop new features to support IPv4/IPv6 dual-stack, ambient mesh, and earlier versions of kernel.
+I look forward to a glorious Merbridge that can be used widely and can really help you to grow.
```
Comment (Michelle951) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
I look forward to a glorious Merbridge that can be used more widely and really help you guys to grow.
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+![Performance comparison between eBPF and iptables for Kuma 2.0](./imgs/kuma-2.0-ebpf-vs-iptables.png)
+
+A quote from [Kuma 2.0 release blog](https://kuma.io/blog/2022/kuma-2-0-0/):
+
+> We are utilizing the Merbridge OSS project within our eBPF capabilities and are very excited
+> that we have been able to contribute back to that library and become co-maintainers. We look forward to working more
+> with the Merbridge team as we continue to explore different areas to include eBPF functionality in Kuma.
+
+As a developer of Merbridge, I'm very glad that Merbridge empowers service mesh providers like Kuma.
```
Comment (hanxiaop) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
As an open source project, we are very excited to see that Merbridge brings such capabitilities to Kuma.
```

File: content/en/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+This case proves that traffic latency can be reduced without any extra overhead if you use Merbridge in a service mesh.
+
+Since June this year, Kuma developers have been working on integrating with Merbridge, trying to get the eBPF-based acceleration capabilities.
+
+Thanks to the clear architecture of Merbridge, Kuma is smoothly adapted with Merbridge in days.
+A big thanks to the Kuma community for contributing such an important compatibility capability to Merbridge, which helps both communities grow together!
+
+So far, Merbridge has the capabilities to support popular service mesh products like Istio, Linkerd2, and Kuma,
+and also has a clear plan to develop new features to support IPv4/IPv6 dual-stack, ambient mesh, and earlier versions of kernel.
+I look forward to a glorious Merbridge that can be used more widely and really help you to grow.
```
Comment (hanxiaop) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md:34]: It is exciting to see that Merbridge gets used more widely. We really hope the project can help you land your project with eBPF technologies. We are looking forward to receiving more comments, and having more developers get involved.

Comment (hanxiaop) [inline content/en/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
It is exciting to see that Merbridge gets used more widely. We really hope the project can help you land your project with eBPF technologies. We are looking forward to receiving more comments, and having more developers get involved.
```

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**